silasdk/client.py

```diff
@@ -11,113 +11,119 @@
 # basic client for making http requests like post,get etc
 
 class App():
-    
-    
-    def __init__(self,tier,app_private_key,app_handle):
+
+    def __init__(self, tier, app_private_key, app_handle):
         """Initalize the application 
             This lets users initialize the application by providing the tier, application privatekey and application handle
         Args:
             tier  : SANDBOX,PROD etc
             app_private_key : ethereum privat key for the application
             app_handle  : application sila handle (app.silamoney.eth)
         """
-        self.session=requests.Session()
-        self.tier=tier.lower()
-        self.app_private_key=app_private_key
-        self.app_handle=app_handle
+        self.session = requests.Session()
+        self.tier = tier.lower()
+        self.app_private_key = app_private_key
+        self.app_handle = app_handle
         self.updateSchema()
 
-        
-    
     def updateSchema(self):
         """updates schema.py on initialization of app
             This lets users initialize the schema into schema.py for ease of use
         Args:
             None
         """
-        endpoint=endPoints["schemaUrl"]
-        message=["header","issue","redeem","transfer","entity","identity","crypto","linkAccount"]
+        endpoint = endPoints["schemaUrl"]
+        message = ["header", "issue", "redeem", "transfer", "entity", "identity", "crypto", "linkAccount"]
         for i in message:
-            response=self.get(
-                    endpoint %i)
-            Schema.append(response)
-        
-            
+            response = self.get(
+                endpoint % i)
+
+            sch = {response["message"]: response}
 
```

Comment:
@csosa-digitalgeko  Can you add the new messages, get wallet etc into this message array, the idea was to cache the latest schema changes not hardcode it, also add a way to clear the schema list on every app initialization, adding Schema.clear() in the update schema function

Reply:
DeepanshuTyagi I have spoken with Avery about the issue of the message, we agree it can stay as it is, for the schema.clear(), I have added now, thanks for your feedback!